implementation/eOrdering/README.txt

```diff
@@ -0,0 +1,21 @@
+Description: The eOrdering module contains classes specific to the Ordering phase of procurement. 
+
+Files:
+The conventions_report file contains the UML Conventions Comformance Report of eOrdering (as an HTML page). 
+An automatically generated report that states the UML conformance violations of the UML model in file 
+eOrdering.xml.
+
+The model2owl-config file contains the configuration files necessary for the model2owl toolchain
+(https://github.com/OP-TED/model2owl) to transform the eOrdering.xml file to a formal OWL ontology
+including SHACL shapes.
+
+The owl_ontology file contains the eOrdering ontology files, as well as the restriction files. The files
+are available in both RDF (https://www.w3.org/RDF/) and Turtle (https://www.w3.org/TR/turtle/) formats. 
+
+The shacl_shapes file contains the SHACL (https://www.w3.org/TR/shacl/) shapes of eOrdering in RDF and 
+Turtle format. SHACL ( Shapes Constraint Language) is a W3C standard used for validating the contents of
+an RDF graph. 
+
+The xmi_conceptual_model file contains the eOrdering.xml file. A representation of the UML model of the 
```

Comment:
just to be consistent with links to turtle, shacl, a link to XMI could be useful